src/main/java/nextstep/fp/Lambda.java

```diff
@@ -26,28 +26,10 @@ public void run() {
         }).start();
     }
 
-    public static int sumAll(List<Integer> numbers) {
+    public static int sumAll(List<Integer> numbers, Conditional conditional) {
         int total = 0;
         for (int number : numbers) {
```

Comment:
해당 반복문도 stream을 이용하여 람다식으로 변경이 가능할 것 같아요 :)
```suggestion
        return numbers.stream()
                      .filter(conditional::filter)
                      .reduce(0, Integer::sum);
```